spec/models/country_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe Country, type: :model do
+  describe 'database' do
+    it 'has columns' do
+      expect(subject).to have_db_column(:name).of_type(:string).with_options(null: false)
+    end
+  end
+
+  describe 'validations' do
+    it { is_expected.to validate_presence_of(:name) }
+
+    context 'when validating unique values' do
```

Comment:
Not sure if we need this context. WDYTA  just keeping the `it` block?
```ruby
describe 'validations' do
  it { is_expected.to validate_presence_of(:name) }
  it { is_expected.to validate_uniqueness_of(:name) }
end
```

Reply:
I think the context was not needed but it was needed to avoid the error of `Shoulda::Matchers::ActiveRecord::ValidateUniquenessOfMatcher::ExistingRecordInvalid:`.

Updated.